lib/util/git/config.ts

```diff
@@ -0,0 +1,11 @@
+export type GitNoVerifyOption = 'commit' | 'push';
+
+let noVerify: GitNoVerifyOption[] = [];
```

Comment:
```suggestion
let noVerify: GitNoVerifyOption[] = [ 'commit', 'push' ];
```

Use default values, so tests behave as if the default config is passed.